index.ts

```diff
@@ -182,6 +182,11 @@ collect.set('isProject', [
 	'https://github.com/sindresorhus/refined-github/projects/3',
 ]);
 
+export const isProjects = (url: URL | HTMLAnchorElement | Location = location): boolean => getRepo(url)?.path === 'projects';
+collect.set('isProjects', [
+	'https://github.com/sindresorhus/refined-github/projects',
```

Comment:
What if it's paged? Can you find a real url for this and add a test for it? You can create them in our sandbox repo if they're allowed

Reply:
> What if it's paged? Can you find a real url for this and add a test for it?

I'm not sure what do you mean... I can only find https://github.com/NixOS/nixpkgs/projects with 28 projects in one page.

> You can create them in our sandbox repo if they're allowed

It needs to be enabled first

Reply:
> I'm not sure what do you mean... I can only find [NixOS/nixpkgs/projects](https://github.com/NixOS/nixpkgs/projects) with 28 projects in one page.
> 
> >

There must be a limit, and after that they'll paginate it. We need to ensure that the test also works on these pages. For example, if the URL is `projects/page/2` then the test will fail.

Reply:
Projects page only shows first 30 entries and the rest is loaded via "Load more..." button:

https://github.com/kidonng/refined-github/projects

Reply:
Thanks for checking!